src/SFA.DAS.EAS.Portal/DependencyResolution/HashingServices.cs

```diff
@@ -0,0 +1,20 @@
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using SFA.DAS.EAS.Portal.Configuration;
+using SFA.DAS.HashingService;
+
+namespace SFA.DAS.EAS.Portal.DependencyResolution
+{
+    public static class HashingServices
```

Comment:
Good tidy up.

Reply:
Thanks! It was a delight to find the `HostBuilderContext`, as it solves a lot of messy workarounds.